aliases/available/systemd.aliases.bash

```diff
@@ -3,13 +3,21 @@ about-alias 'systemd service'
 
 case $OSTYPE in
     linux*)
+# Improve aliases by bringing the common root `sc|scd` + `sre` for action + `u` for user
 	alias sc='systemctl'
-	alias scr='systemctl daemon-reload'
 	alias scu='systemctl --user'
-	alias scur='systemctl --user daemon-reload'
+	alias scdr='systemctl daemon-reload'
+	alias scdru='systemctl --user daemon-reload'
+	alias scr='systemctl restart'
+	alias scru='systemctl --user restart'
 	alias sce='systemctl stop'
-	alias scue='systemctl --user stop'
+	alias sceu='systemctl --user stop'
 	alias scs='systemctl start'
+	alias scsu='systemctl --user start'
+# Keeping previous aliases for a non-breaking change.
+	alias scue='systemctl --user stop'
 	alias scus='systemctl --user start'
+	alias scr='systemctl daemon-reload'
```

Comment:
this will override the prev aliased `scr` in line 11, so I guess we will deprecate it completely and remove this line

Reply:
Thanks for catching this, @NoahGorny, removing this one.